<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioInput</class>
 <widget class="QMainWindow" name="AudioInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AudioInput</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Read Method</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <item>
        <widget class="QRadioButton" name="rbtDevice">
         <property name="text">
          <string>IO Device</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbtQFile">
         <property name="text">
          <string>QFile</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBoxDevice">
         <property name="title">
          <string>support device</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="8" column="0" colspan="2">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Codec</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Channels</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Endianness</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0" colspan="2">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="cbSampleType">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cbFrequency">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>SampleType</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="labelVolume">
            <property name="text">
             <string>volume(100%)</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QComboBox" name="cbEndianess">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="cbChannels">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>SampleSize</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbEquip">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cbCodec">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="2">
           <widget class="QLabel" name="label_8">
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="text">
             <string>8000Hz 1 channel audio/pcm 8bits UnSignedInt LittleEndian</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QSlider" name="SliderVolume">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Frequency(Hz)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="cbSampleSize">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Input Equipment</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxBuffer">
         <property name="title">
          <string>buffer info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="9" column="0">
           <widget class="QLabel" name="labelIODevice">
            <property name="text">
             <string>IODevice:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0" columnstretch="2,1,2,0">
             <item row="0" column="1">
              <widget class="QProgressBar" name="pbMin">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="inputMethodHints">
                <set>Qt::ImhDigitsOnly</set>
               </property>
               <property name="value">
                <number>50</number>
               </property>
               <property name="textVisible">
                <bool>false</bool>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="textDirection">
                <enum>QProgressBar::TopToBottom</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QProgressBar" name="pbMax">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="inputMethodHints">
                <set>Qt::ImhDigitsOnly</set>
               </property>
               <property name="value">
                <number>50</number>
               </property>
               <property name="textVisible">
                <bool>false</bool>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="textDirection">
                <enum>QProgressBar::TopToBottom</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QProgressBar" name="pbDifference">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="inputMethodHints">
                <set>Qt::ImhDigitsOnly</set>
               </property>
               <property name="value">
                <number>50</number>
               </property>
               <property name="textVisible">
                <bool>false</bool>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="textDirection">
                <enum>QProgressBar::TopToBottom</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>max</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>min</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>max-min</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="labelBufferSize">
            <property name="text">
             <string>bufferSize:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="labelBytesReady">
            <property name="text">
             <string>bytesReady:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>505</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRecord"/>
   <addaction name="actionStart"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRecord">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/5.ico</normaloff>:/images/5.ico</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/622.bmp</normaloff>:/images/622.bmp</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/624.bmp</normaloff>:/images/624.bmp</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/132.bmp</normaloff>:/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/626.bmp</normaloff>:/images/626.bmp</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>AudioInput</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
